setup.cfg

```diff
@@ -1,3 +1,78 @@
+[metadata]
+name = photutils
+provides = photutils
+version = 0.7.dev
+author = Photutils Developers
+author_email = photutils.team@gmail.com
+license = BSD 3-Clause License
+license_file = LICENSE.rst
+url = https://github.com/astropy/photutils
+github_project = astropy/photutils
+edit_on_github = False
+description = An Astropy package for source detection and photometry
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+keywords = astronomy, astrophysics, photometry, aperture, psf, source detection, background, segmentation, centroids, isophote, morphology
+classifiers =
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Cython
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Astronomy
+
+[options]
+python_requires = >=3.5
+packages = find:
+setup_requires =
+    numpy>=1.13
+    cython>=0.28
+install_requires =
+    numpy>=1.13
+    astropy>=2.0
+tests_require =
+    pytest
+    pytest-astropy
```

Comment:
Ok I thought this bug seemed familiar. The issue is that cython can segfault if psutil is installed. Don't ask... (@bsipocz @pllim - do you know if we had an issue for that somewhere? I know we've discussed it but is there an issue?)

The workaround is to not add pytest-astropy here, but pytest-remotedata and pytest-doctestplus, as it's pytest-openfiles that drags in psutils.

Reply:
Hooray!  Huge thanks!  :tada:

Does `cython` segfault with `psutil` only for python 3.5?

Reply:
It seems so yes.

Reply:
This vaguely rings a bell, but I don't think we have an issue for it. Is it an issue with a particular psutils version? If yes, maybe we can pin the version in `pytest-astropy` or in `pytest-openfiles`?

Reply:
Time to drop python 3.5.... 😈

Reply:
core dropped it, so it's a reasonable move. (hurray f-strings)

Reply:
I searched in several repo of interest but can't find the issue either. 🤷‍♀

Reply:
Indeed, f-strings are very nice.  I'll keep py3.5 for this release, but drop it for v0.8.